### ISSUE TITLE
custom model 

### ISSUE THREAD

**Issue by Rawabi20** [zezo357/pytorch_lite#71]
I want to use my own yolo model it is for sign language detection..
I tried to just replace the models and labels paths but nothing appear.. No errors and also no label or bounding box.
I don't know if there is something else needs to be changed
This how I change the code:
<img width="960" alt="image" src="https://github.com/zezo357/pytorch_lite/assets/131215231/9813ac12-a3de-4e28-835d-7f5122b2f9f9">
here is what is the terminal keep showing:
<img width="714" alt="image" src="https://github.com/zezo357/pytorch_lite/assets/131215231/e8c75b5a-d27b-4c98-a945-c7302a48fcd4">

**Comment by zezo357** [comment]
Your model didn't detect any objects, the before processing part is all of the objects detected





Check your weights and the way you exported your model

**Comment by Rawabi20** [comment]
> Your model didn't detect any objects, the before processing part is all of the objects detected
> 
> Check your weights and the way you exported your model

How can i train the model? I have the imgs with their annotations and apply this python code to get .pt file
<img width="664" alt="image" src="https://github.com/zezo357/pytorch_lite/assets/131215231/6cd515d6-c698-4e24-8b16-b42bcab9803e">
after that I convert best.pt into .torchscript using the line you provided
`!yolo mode=export model="your model" format=torchscript optimize`

**Comment by zezo357** [comment]
i dont see a problem with your code, can you try another model or other images

**Comment by Rawabi20** [comment]
> i dont see a problem with your code, can you try another model or other images

I have tried another model and its worked as expected, but the real time detection is too slow is their any way to improve it?

**Comment by zezo357** [comment]
Sadly for now there is none, it's a limitation from pytorch it self, for the best speeds you will have to use tflite

**Comment by Rawabi20** [comment]
> Sadly for now there is none, it's a limitation from pytorch it self, for the best speeds you will have to use tflite

I already tried it but it is requires flutter downgrade ,because it is not work with the latest flutter version 😢

**Comment by zezo357** [comment]
Sadly the package to the latest pytorch and no GPU support yet😢

**Comment by nqtworking** [comment]
Hi @zezo357 , I'm a beginner in computer vision and Flutter. I'm wondering if your pre-made example has the ability to convert images from the camera to int8 format and then pass them to a YOLO model exported as int8 to improve processing speed?

**Comment by zezo357** [comment]
The example does convert camera image to it's float32 format since that's the input for yolo yes

**Comment by nqtworking** [comment]
@zezo357 thanks for your feedback!
